### DISCUSSION TITLE
Integration with Petals

### DISCUSSION THREAD

**Discussion by RUNdjc0** [oobabooga/text-generation-webui#3387] [Ideas]
For those of us with little cash or GPUs. Pretty please with sugar on top?

**Comment by shohamjac** [comment]
I took it as a project. Guess I will publish a basic version in a week.

**Comment by Kalicody** [comment]
any update with Petals integration ? 

best regards,

**Comment by shohamjac** [comment]
yea @mathnerd314 beat me to it. 
He pushed a PR [here](https://github.com/oobabooga/text-generation-webui/pull/3784).
